gqcp/tests/QCMethod/CC/CCD0_test.cpp

```diff
@@ -0,0 +1,257 @@
+#define BOOST_TEST_MODULE "CCD0"
+
+#include <boost/test/unit_test.hpp>
```

Comment:
Please add the header files from ‘more specific’ to ‘more general’.

This means
- Our includes first
- Boost/Eigen headers
- STL headers


Unless it’s headers for `boost/test`, these should always go first because Libint (one of our dependencies otherwise starts complaining).